tasks/lzards-backup/src/errors.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line max-classes-per-file
+export class CollectionError extends Error {
```

Comment:
Giving this just a few moments of thought, I'd expect that *generally* you want to use `errors` in case of a re-usable error that is going to be caught/checked/etc across multiple core packages.     Given this is the top level of a task, I wouldn't expect anything in Core to be importing/checking for these, however I suppose I could pull in the generator code to avoid the boilerplate.

Reply:
I'm indifferent to whether we use the `createErrorType` helper or not, though I suppose it seems a bit streamlined